lib/python-parser/metadata-parser.ts

```diff
@@ -29,7 +30,17 @@ export function getPackageInfo(fileContent: string): PythonPackage {
       }
     }
   }
-  return { name: name.toLowerCase(), version, dependencies } as PythonPackage;
+  const validVersion = semver.coerce(version);
+  if (!validVersion) {
+    throw new PythonInvalidVersionError(
+      `version ${version} is not compatible with semver and cannot be compared`,
+    );
+  }
+  return {
+    name: name.toLowerCase(),
+    version: validVersion ? validVersion.toString() : version,
```

Comment:
I guess we don't need that check then, if we throw an error :)

Reply:
oh, you're right. forgot to delete